pulsar-broker/src/test/java/org/apache/pulsar/broker/service/PersistentTopicTest.java

```diff
@@ -249,20 +252,14 @@ public void setup() throws Exception {
 
     @AfterMethod(alwaysRun = true)
     public void teardown() throws Exception {
-        metadataStore.close();
```

Comment:
is it fine to replace this with `store.close()`?

Reply:
yes it's intentional. `store` is created by `PersistentTopicTest`. `metadataStore` is handled by the superclass `MockedBookKeeperTestCase`. So before the `store` was never closed

Reply:
it seems that there's both store and metadataStore (defined in MockedBookKeeperTestCase). I think that metadataStore state should be cleaned up in MockedBookKeeperTestCase.tearDown method.

Reply:
@nicoloboschi Good catch!